strings.txt

```diff
@@ -813,16 +822,17 @@ PLUGIN_SPOTTY_CLIENT_ID_DESC
 	NL	<div>Het gebruik van een aangepaste Client ID kan helpen als er problemen zijn met de snelheidsbeperking ("error 429"). Volg deze eenvoudige stappen om een eigen klant-ID te krijgen:</div>
 	NL	<ul style="list-style:initial">
 	NL	<li>Ga naar <a href="https://developer.spotify.com/dashboard/applications" target="_blank">de Spotify Developer Portal</a>, log indien nodig in.</li>
-	NL	<li>Klik op "Create An App", volg de instructies.</li>
+	NL	<li>Klik op "Create app", volg de instructies.</li>
 	NL	<li>Kopieer de nieuw aangemaakte Client ID naar jouw Spotty instellingen.</li>
 	NL	<li>Als je van plan bent meerdere Spotify accounts met Spotty te gebruiken (bijv. een gezinsaccount), moet je de accounts toevoegen aan het gedeelte 'Gebruikers en toegang' in de nieuw gemaakte configuratie.</li>
 	NL	</ul>
 
 PLUGIN_SPOTTY_ADD_USER_TO_CLIENT_ID
+	CS	Protože používáte vlastní Client ID, budete muset přejít na <a href="https://developer.spotify.com/dashboard/applications" target="_blank">Spotify Developer Portal</a>, v případě potřeby se přihlásit a přidat účty do sekce „User Management“.
```

Comment:
Where did you find the "User Management" section? Can't find it any more 😬.

Reply:
I think this is the section
<img width="1300" height="814" alt="image" src="https://github.com/user-attachments/assets/ccee5505-4dec-4146-8bae-9a29ee433477" />

Reply:
Heh... I'd have an "Extension Requests" tab there, but no "User management". That's for my main client ID. I have three more, one of which doesn't have any additional tab, and the others indeed have "User Management". I didn't look further when I didn't see this in the first (main) entry. Thanks for the clarification!